change/@ni-nimble-components-56124785-d76b-4bd0-b330-b457eb4ff9a0.json

```
@@ -1,6 +1,6 @@
 {
   "type": "major",
-  "comment": "Adding eight icons, renaming two, and replacing two",
+  "comment": "Adding eight icons, renaming two, and replacing two. Clients using \"nimble-icon-calendar\" or \"nimble-icon-arrow-u-rotate-left\" must instead use \"nimble-icon-calendar-days\" and \"nimble-icon-arrow-u-up\" (respectively).",
   "packageName": "@ni/nimble-components",
   "email": "7282195+m-akinc@users.noreply.github.com",
   "dependentChangeType": "patch"
```

Clients will mostly be using the dependent libraries nimble-angular or NimbleBlazor and it'd be good to signal the breaking change there too.
```suggestion
  "dependentChangeType": "major"
```